src/v3/src/transformer/selectAuthenticator/utils.ts

```diff
@@ -251,13 +251,17 @@ const formatAuthenticatorOptions = (
         AUTHENTICATOR_KEY.PHONE,
         AUTHENTICATOR_KEY.CUSTOM_APP,
       ];
+      const AUTHENTICATORS_WITH_LTR_DESCRIPTION = [
+        AUTHENTICATOR_KEY.PHONE,
+      ];
       const authenticator = option.relatesTo;
 
       return {
         type: 'AuthenticatorButton',
         label: option.label,
         id: `auth_btn_${authenticatorKey}_${enrollmentId || id}`,
         noTranslate: !isEnroll && AUTHENTICATORS_WITH_NO_TRANSLATE_CLASS.includes(authenticatorKey),
+        dir: !isEnroll && AUTHENTICATORS_WITH_LTR_DESCRIPTION.includes(authenticatorKey) ? 'ltr' : undefined,
```

Comment:
FYI there is a `uischema/setLtrFields` transformer but it may not make sense. Just wanted to make sure you were aware of it and considered it.